src/data/Dhis2RelationshipTypes.ts

```diff
@@ -239,7 +239,7 @@ async function getConstraintForTypeTei(
 
     const ouModeQuery =
         ouMode === "SELECTED" || ouMode === "CHILDREN" || ouMode === "DESCENDANTS"
-            ? { orgUnitMode: ouMode, orgUnit: organisationUnits?.map(({ id }) => id) }
+            ? { orgUnitMode: ouMode, orgUnit: organisationUnits?.map(({ id }) => id).join(";") }
```

Comment:
This code 
```ts
organisationUnits?.map(({ id }) => id).join(";")
```

is being used in the file `src/data/Dhis2TrackedEntityInstances.ts` as well. Can we create a function `buildOrgUnitParameter` or something similar and reuse it?

```ts
function buildOrgUnitParameter(orgUnitsIds: Maybe<Id[]>): string {
 return orgUnitsIds?.map(({ id }) => id).join(";")
}
```

Reply:
okay got it!